synapse/handlers/federation.py

```diff
@@ -288,19 +305,6 @@ def _get_missing_events_for_pdu(self, origin, pdu, prevs, min_depth):
                 get_missing=False
             )
 
-        have_seen = yield self.store.have_events(
-            [ev for ev, _ in pdu.prev_events]
-        )
-        seen = set(have_seen.keys())
-        if prevs - seen:
-            logger.info(
-                "Still missing %d prev events for %s: %r...",
-                len(prevs - seen), pdu.event_id, list(prevs - seen)[:5]
-            )
-        else:
-            logger.info("Found all missing prev events for %s", pdu.event_id)
-        defer.returnValue(have_seen)
```

Comment:
there are other calls to `defer.returnValue` in this function; please update them too

also please update the docstring.